.github/workflows/main.yml

```diff
@@ -79,8 +78,8 @@ jobs:
       - name: Install dependencies
         run: poetry install
 
-      - name: Install pytest plugin
-        run: poetry run pip install pytest-github-actions-annotate-failures
-
       - name: Run pytest
-        run: poetry run python -m pytest -p no:sugar -q tests/
+        run: poetry run python -m pytest -vv --color=yes -p no:clarity -p no:sugar --cov=src/poetry --cov-report xml tests/
```

Comment:
we can add coverage run config and pytest args into `pyproject.toml` right?

Reply:
Yes, but it significantly increases the execution time of the test suite for local testing, so I feel that it's better done here. Also, we only want to run coverage on Github Actions as Codecov does not currently have an uploader that runs on FreeBSD.

(it's more complicated than that -- technically we can install node.js and their typescript uploader on FreeBSD -- there's just no ready-to-run bundle)